src/brands.ts

```diff
@@ -0,0 +1,63 @@
+import { 
+  ICourierBrand, 
+  ICourierBrandGetAllResponse,
+  ICourierBrandPostParameters,
+  ICourierBrandPutParameters,
+  ICourierClientConfiguration
+} from "./types";
+
+export const getBrands = (options: ICourierClientConfiguration) => {
+  return async (
+    params?: {
+      cursor: string;
+    }
+  ): Promise<ICourierBrandGetAllResponse> => {
+    const res = await options.httpClient.get<ICourierBrandGetAllResponse>(
+      `/brands`, params
+    );
+    return res.data;
+  };
+};
+
+export const getBrand = (options: ICourierClientConfiguration) => {
+  return async (
+    brandId: string
+  ): Promise<ICourierBrand> => {
+    const res = await options.httpClient.get<ICourierBrand>(
+      `/brands/${brandId}`
+    );
+    return res.data;
+  };
+};
+
+export const createBrand = (options: ICourierClientConfiguration) => {
+  return async (
+    params: ICourierBrandPostParameters
+  ): Promise<ICourierBrand> => {
+    const res = await options.httpClient.post<ICourierBrand>(
+      `/brands`,
+      params
+    );
+    return res.data;
+  };
+};
+
+export const replaceBrand = (options: ICourierClientConfiguration) => {
+  return async (
+    params: ICourierBrandPutParameters
+  ): Promise<ICourierBrand> => {
+    const {id, ...rest} = params;
+    const res = await options.httpClient.put<ICourierBrand>(
+      `/brands/${id}`,
+      rest
+    );
+    return res.data;
+  };
+};
+
+export const deleteBrand = (options: ICourierClientConfiguration) => {
+  return async (brandId: string): Promise<object> => {
```

Comment:
`object` is almost as vague as `any`. Is there a type we can substitute here?

Reply:
I think this is where I was like `any` isn't good. Should I make it `object`? But this could be `void` or something else. I'm not sure.